### DISCUSSION TITLE
Suggestion: Have "zoom to fit" include document shadow and/or make document shadow user configurable

### DISCUSSION THREAD

**Discussion by Dennisman219** [flxzt/rnote#218] [Ideas]
First off, I absolutely love this app.

I mainly use rnote with the 'continuous vertical' document layout. I like to use the 'zoom to fit' function in other apps to zoom to the level where there is no horizontal scrolling possible.

The 'zoom to fit' function in rnote does increase the size of the document to the available horizontal width, but the hardcoded `Document::SHADOW_WIDTH` creates a margin around the document which results in still being able to scroll horizontally.

I would suggest that the 'zoom to fit' button includes the shadow size in order to zoom to the level where there is no horizontal scrolling possible.

For users with small screens/devices (such as myself) it would also be helpful if the shadow size could be adjusted to allow them to increase the size of the document relative to the available realestate.

**Comment by flxzt** [comment]
Sounds like a good improvement. Including the shadow width was an easy fix. Variable shadow width will require more effort.
